mutacc/utils/bam_handler.py

```diff
@@ -198,84 +202,51 @@ def make_names_temp(self, out_dir):
         return self.names_temp
 
 
-def check_bam(bam_file):
+    def _check_bam(self):
 
-    """Checks if reads in bam are paired
-    Args:
-        bam_file (path): path to bam file
+        """Checks if reads in bam are paired
+        Args:
+            bam_file (path): path to bam file
 
-    Returns:
-        paired (bool): True if reads are paired
-    """
-    paired = False
-    with pysam.AlignmentFile(bam_file) as bam:
+        Returns:
+            paired (bool): True if reads are paired
+        """
+
+        insert_sizes = []
+        acc_length = 0
+        paired = False
         count = 0
-        for read in bam:
+        no_reads = 10000
```

Comment:
```suggestion
        MAX_READ_COUNT = 10000
```
It is a constant after all 😄